PyRDF/backend/Utils.py

```diff
@@ -1,10 +1,41 @@
-import ROOT
-import os
 import logging
+import multiprocessing
+import os
+import threading
+
+try:
+    import queue
+except ImportError:
+    import Queue as queue
+
+import ROOT
 
 logger = logging.getLogger(__name__)
 
 
+def run_graphs(proxies, numthreads=multiprocessing.cpu_count()):
+    """
+    Trigger multiple RDF graphs through multithreading.
+    """
+    ROOT.EnableThreadSafety()
+
+    q = queue.Queue()
+
+    for proxy in proxies:
+        q.put(proxy)
+
+    def trigger_loop(queue_):
+        while True:
+            queue_.get().GetValue()
+            queue_.task_done()
```

Comment:
Does this `task_done` has a "thread local" effect?

Reply:
Without it the queue will not unblock the next item to process, but there's no other effect on the items themselves. More info at https://docs.python.org/3/library/queue.html#queue.Queue.task_done